kie-drools-wb-parent/kie-drools-wb-webapp/src/main/webapp/rest-api.jsp

```diff
@@ -41,19 +41,18 @@ Entry point for REST interface is at <%= request.getScheme()+"://"+request.getSe
   <tr><td>/organizationalunits/{organizationalUnitName}</td><td>GET</td><td>get the details for an organizational unit</td></tr>
```

Comment:
All endpoints for work with one org unit looks like /organizationalunits/{organizationalUnitName} expect the one for create an org unit. We should change this create org unit REST endpoint.